groovy/material/FirstDegreeMaterials.groovy

```diff
@@ -553,7 +559,11 @@ public class FirstDegreeMaterials {
                 .colorAverage()
                 .build();
 
-        //FREE ID: 8170
+        CopperISulfide = new Material.Builder(8170, SuSyUtility.susyId('copper_i_sulfide'))
+                .dust()
+                .components(Copper, 2, Sulfur, 1)
```

Comment:
For some reason, the "2" in the formula does not appear in-game. Perhaps try to set the formula?